pkg/api/ldap.go

```diff
@@ -183,26 +186,28 @@ func (lc *LDAPClient) Authenticate(username, password string) (bool, map[string]
 		lc.Log.Error().Err(err).Str("bindDN", lc.BindDN).Str("username", username).
 			Str("baseDN", lc.Base).Msg("search failed")
 
-		return false, nil, err
+		return false, nil, nil, err
 	}
 
 	if len(search.Entries) < 1 {
 		err := errors.ErrBadUser
 		lc.Log.Error().Err(err).Str("bindDN", lc.BindDN).Str("username", username).
 			Str("baseDN", lc.Base).Msg("entries not found")
 
-		return false, nil, err
+		return false, nil, nil, err
 	}
 
 	if len(search.Entries) > 1 {
 		err := errors.ErrEntriesExceeded
 		lc.Log.Error().Err(err).Str("bindDN", lc.BindDN).Str("username", username).
 			Str("baseDN", lc.Base).Msg("too many entries")
 
-		return false, nil, err
+		return false, nil, nil, err
 	}
 
 	userDN := search.Entries[0].DN
+	userAttributes := search.Entries[0].Attributes[0]
+	userGroups := userAttributes.Values
```

Comment:
There will be the FQDNs of the groups, right? Is there a way to map this to the user groups in the zot config? Or the user needs to use the FQDN value in all the authz policies, without needing additional mapping to a more friendly name in the config?